activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/EventsRestTemplate.java

```diff
@@ -56,7 +56,7 @@ public EventsRestTemplate(ObjectMapper mapper, KeycloakTokenProducer keycloakTok
     }
 
     @Bean
-    public RestTemplateBuilder restTemplateBuilder(List<Module> modules) {
+    public RestTemplateBuilder restTemplateBuilder2(List<Module> modules) {
```

Comment:
I guess this was a temporary renaming, right?

Reply:
yes, due to problem in compilation, fixing

Reply:
Maybe it is better to merge and handles the new change to the database with liquibase, adding a new changeset, WDYT?